nasa-gibs/worldview
# WV-2887 Layer Classifications Toggle Updating Issue

## Description
This adds preloading to all the imagery in a tour story when a tour story is initiated. Custom palettes are implemented. Still to be implemented: min/max palette values.

## How To Test
1. `git checkout wv-2887-preload-tourstory`
2. `npm ci`
3. `npm run watch`
4. Open any Tour Story and navigate through each step.
5. Verify that the imagery for each step is already preloaded before being the step is rendered onscreen.
